tests/integration/local/invoke/test_integrations_cli.py

```diff
@@ -94,3 +94,14 @@ def test_invoke_when_function_writes_stderr(self):
         process_stderr = b"".join(process.stderr.readlines()).strip()
 
         self.assertIn("Docker Lambda is writing to stderr", process_stderr.decode('utf-8'))
+
+    def test_invoke_raises_exception_with_noargs_and_event(self):
```

Comment:
Can you add a small integ test for providing `--no-agrs` and making sure the function invoked and returns as expected?

Reply:
Will do.